crates/common_enums/src/enums.rs

```diff
@@ -1434,9 +1434,8 @@ impl From<AttemptStatus> for PaymentMethodStatus {
             | AttemptStatus::PartialCharged
             | AttemptStatus::PartialChargedAndChargeable
             | AttemptStatus::ConfirmationAwaited
-            | AttemptStatus::DeviceDataCollectionPending
```

Comment:
we also have `processing` status in payment method status. What does it mean? can it be mapped to some of the processing attempt statuses?